prime-router/src/main/kotlin/azure/BatchDeciderFunction.kt

```diff
@@ -28,27 +27,27 @@ class BatchDeciderFunction : Logging {
     ) {
         logger.info("$batchDecider: Starting")
         try {
-            val db = WorkflowEngine().db
-            db.transact { txn ->
+            val workflowEngine = WorkflowEngine()
+            workflowEngine.db.transact { txn ->
                 // TODO: if for some reason the batch decider misses proper calculation of a receiver's batch run
                 //  we would want to pull all receivers with BATCH records that have next_action_at in the past and
                 //  merge those with the batchInPrevious60Seconds. Testing shows this to be an unlikely scenario, but
                 //  there is always the chance that something odd happens with the timing of Azure timed functions.
 
                 // find all receivers that should have batched within the last 60 seconds
-                settings.receivers.filter { it.timing != null && it.timing.batchInPrevious60Seconds() }
```

Comment:
This settings was pointing to obj in TestReportStream which uses the FileSettings.   So this was using local settings even in Staging.